src/twisted/internet/task.py

```diff
@@ -900,10 +900,11 @@ def react(
 
     The following demonstrates the signature of a C{main} function which can be
     used with L{react}::
-          async def main(reactor, username, password):
-              return "ok"
 
-          task.react(main, ("alice", "secret"))
+      async def main(reactor, username, password):
+          return "ok"
+
+      task.react(main, ("alice", "secret"))
```

Comment:
View this here: https://twisted--11598.org.readthedocs.build/en/11598/api/twisted.internet.task.html